src/components/course/CourseCapturePreferences.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div>
+    <section :aria-labelledby="'future-courses-header'">
+      <h2 id="future-courses-header" class="mt-3 text-medium-emphasis w-100">
+        Future courses
+      </h2>
+      <div class="mt-2 px-md-4 w-100">
+        <v-radio-group
+          v-model="futureCoursesPref"
+          :aria-labelledby="'future-courses-header'"
```

Comment:
@mohamalnadi here and elsewhere, I don't think it's necessary to make the aria-labelledby dynamic if the value is just a string. as in, you can write `aria-labelledby="future-courses-header"` rather than `:aria-labelledby="'future-courses-header'"`.